tests/conftest.py

```diff
@@ -102,7 +102,7 @@ def compileSolidity(relativeFilePath):
                 }
             },
             'settings': {
-                'remappings': [ 'ROOT=%s' % resolveRelativePath("../source/contracts") ],
+                'remappings': [ '=%s/' % resolveRelativePath("../source/contracts") ],
```

Comment:
We should be able to remove this line and edit line 100 above to change the key to be the relative file path (no prefix) just like you did in the new compile stuff.  Something like `stripPrefix(resolveRelativePath("../source/contracts/"), absoluteFilePath)` (made up some of those functions but that is the idea).

If that proves to be too difficult and this works, we can leave it like this for now and make that change later.

Reply:
Yeah, I figured this line should be taken out altogether, but removing it didn't work, and I wasn't sure exactly what other change I needed to make.  I'll update the path in the `sources` array like you mentioned.

Reply:
I tried:
```
resolveRelativePath(relativeFilePath).replace(resolveRelativePath("../source/contracts/") + "/", "")
```
which should remove the prefix and just leave you with "Controller.sol".  However, that didn't work.  Since it's getting late, I think I'll just merge this for now and fix it in the `deployment` branch.